samples/CommunityToolkit.Maui.Sample/Pages/Converters/MultiConverterPage.xaml

```diff
@@ -34,32 +35,32 @@
                     ConverterType="{x:Type mct:TextCaseConverter}"
                     Value="{x:Static mct:TextCaseType.Upper}" />
             </x:Array>
-            
+
             <mct:VariableMultiValueConverter ConditionType="All" x:Key="VariableMultiValueConverterAll" />
             <mct:InvertedBoolConverter x:Key="InvertedBoolConverter" />
         </ResourceDictionary>
     </pages:BasePage.Resources>
-    
+
     <pages:BasePage.Content>
         <VerticalStackLayout Padding="20"
                              Spacing="20">
-            <Label Text="This sample demonstrates how to use the MultiConverter with the IsNotEqualConverter and the TextCaseConverter. It converts the entered text to Upper Case and checks that it is Not Equal to the string 'MAUI'."/>
+            <Label x:DataType="vm:MultiConverterViewModel" Text="This sample demonstrates how to use the MultiConverter with the IsNotEqualConverter and the TextCaseConverter. It converts the entered text to Upper Case and checks that it is Not Equal to the string 'MAUI'."/>
```

Comment:
There is no `Binding` on this `Label`. I don't think we need to set the `x:DataType` here.